src/ObjectPool/ref/Microsoft.Extensions.ObjectPool.netcoreapp.cs

```diff
@@ -13,6 +13,7 @@ public partial class DefaultObjectPool<T> : Microsoft.Extensions.ObjectPool.Obje
     {
         public DefaultObjectPool(Microsoft.Extensions.ObjectPool.IPooledObjectPolicy<T> policy) { }
         public DefaultObjectPool(Microsoft.Extensions.ObjectPool.IPooledObjectPolicy<T> policy, int maximumRetained) { }
+        [return: System.Diagnostics.CodeAnalysis.NotNullAttribute]
```

Comment:
Should NotNull also be on the base class? Would it ever make sense to return null from ObjectPool?

Reply:
It's already on the base class too.

Reply:
I don't see it - https://github.com/dotnet/aspnetcore/blob/42963fbc454e5cc6fedf2389700d73257a97f2a5/src/ObjectPool/ref/Microsoft.Extensions.ObjectPool.netcoreapp.cs#L57-L62

Reply:
Oops, I think I meant to put this on the base type.